docs/platform/concepts/permissions.md

```diff
@@ -39,6 +39,7 @@ and services within it.
 | View organization audit log | `organization:audit_logs:read` | <ul> <li> View the audit log. </li> </ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | Manage domains              | `organization:domains:write`   | <ul> <li> Add, edit, and remove domains. </li> <li> View all organization domains. </li> </ul>                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | Manage groups               | `organization:groups:write`    | <ul> <li> Create, edit, and delete groups. </li> <li> Add organization and application users to groups. </li> <li> Remove organization and application users from groups. </li> </ul>                                                                                                                                                                                                                                                                                                                          |
+| Manage projects             | `organization:projects:write`  | <ul> <li> Create and delete projects. </li> <li> Assign projects to billing groups. </li> <li> Add and remove project tags. </li> <li>  </li> </ul> **Cannot otherwise access or move the project or its services.**                                                                                                                                                                                                                                                                                           |
```

Comment:
This one doesn't render correctly in the GUI. Please see the preview.

Reply:
I guess you meant the typo with the extra bullet point? That's been removed now.

Reply:
![Screenshot 2025-01-30 at 12 57 22](https://github.com/user-attachments/assets/014e7c5a-7573-4352-916e-5e8c035139ee)